### DISCUSSION TITLE
How to move message on Yaml file to the right of lualine

### DISCUSSION THREAD

**Discussion by jorge-ramirezs** [nvim-lualine/lualine.nvim#709] [Q&A] [answered]
Hello everyone,

My question is how can I move the "Yaml No JSON Schema" message to the right? to lualine_x specifically.

I'm using yaml files and every time I open a file I see that message at the left of lualine, even before the mode :

![image](https://user-images.githubusercontent.com/63554124/173409071-2624e86c-f280-49b3-a030-2658a9180cd5.png)

this is my setup. I'm using nvim 0.7

``` vim
lua << EOF
require('lualine').setup {
  options = {
    icons_enabled = true,
    theme = 'auto',
    component_separators = { left = '', right = ''},
    section_separators = { left = '', right = ''},
    disabled_filetypes = {},
    always_divide_middle = true,
    globalstatus = false,
  },
  sections = {
    lualine_a = {'mode'},
    lualine_b = {'branch', 'diff', 'diagnostics'},
    lualine_c = {'filename'},
    lualine_x = {'encoding', 'fileformat', 'filetype'},
    lualine_y = {'progress'},
    lualine_z = {'location'}
  },
  inactive_sections = {
    lualine_a = {},
    lualine_b = {},
    lualine_c = {'filename'},
    lualine_x = {'location'},
    lualine_y = {},
    lualine_z = {}
  },
  tabline = {
    lualine_a = {},
    lualine_b = {},
    lualine_c = { require'tabline'.tabline_buffers },
    lualine_x = { require'tabline'.tabline_tabs },
    lualine_y = {},
    lualine_z = {},
  },
  extensions = {'fugitive', 'nerdtree'}
}
EOF
```

Thank you,
Jorge.

**Comment by shadmansaleh** [comment, marked as answer]
Thats not from lualine . Some other plugin you're using is putting this message in front of statusline . check that plugin if they provide a way to do that.

**Reply by jorge-ramirezs** [reply]
you are right, looks like one of the other plugins I had was putting the message, probably coc but I'm not sure, I started a new config file and now the message has disappeared.

Thank you.
